test/runner.js

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+
+/*eslint no-process-exit:0*/
+'use strict';
+
+//Setup environemnt
+process.env.NODE_ENV = 'test';
+
+if (process.argv.length === 2) {
+  process.argv.push('unit', 'integration');
+}
+
+// Determine which tests to run based on argument passed to runner
+var args = process.argv.splice(2);
+var root;
+var files;
+var runUnitTests = false;
+
+if (args[0] === 'lint') {
+  root = 'test/unit/';
+  files = 'eslint.test.js';
+  runUnitTests = true;
+}
+
+if (args.indexOf('unit') > -1) {
+  root = 'test/unit';
+  files = '/**/*.js';
+  runUnitTests = true;
+}
+
+//Run unit tests
+if (runUnitTests) {
+  var glob  = require('glob');
+  var Mocha = require('mocha');
+
+  //Define Mocha
+  var mocha = new Mocha({
+    // For some reason, tests take a long time on Windows (or at least AppVeyor)
+    timeout: (process.platform === 'win32') ? 30000 : 18000,
+    reporter: 'spec'
+  });
+
+  root = 'test/unit';
```

Comment:
What about `test/integration`, `test/structure` and `test/support`? These are currently being ran as well.